### PR TITLE
[CONTENT] Moonskip death event shortening

### PR DESCRIPTION
**Pull request by scribblecrumb**
## About The Pull Request
A discussion about writing style was brought up in the discord. Consensus seemed to like the idea of keeping most moon events short and sweet, while allowing patrols to have greater detail.  I've started working through the longer moon events to bring them more in line with the discussed style.  This PR just covers the mountain and general death files.  There's also a couple tense fixes sprinkled in (to bring everything into past tense for consistency) and I think one deleted event? It was redundant.

## Why This Is Good For ClanGen
Writing style is nice to be consistent with ;) 

## Proof of Testing
Considering that I only changed json files, as long as the json test passes I think we're fine.

## Changelog/Credits
- Changes to wording of general and mountain-specific death events to bring them more in line with intended writing style.

### CODE REVIEW

File: resources/dicts/events/death/general.json [hunk 1]
```diff
@@ -1116,7 +1116,7 @@
         ],
         "tags": [],
         "weight": 20,
-        "event_text": "m_c has always wondered where {PRONOUN/m_c/subject} came from. {PRONOUN/m_c/subject/CAP} {VERB/m_c/sneak/sneaks} out of camp, intending to find loners who might have some clues about {PRONOUN/m_c/poss} origins. Frantic with worry and confusion after finding m_c's empty nest in the morning, c_n launches a search. They find {PRONOUN/m_c/poss} cooling body after a long search. They wonder why {PRONOUN/m_c/subject} decided to leave the safety of c_n, and what killed {PRONOUN/m_c/object}.",
+        "event_text": "m_c sneaked out of camp, intending to find more about {PRONOUN/m_c/poss} origins. c_n found {PRONOUN/m_c/poss} cooling body in the morning, after a long search.",
```
Comment (Tiritirimatangi) [inline resources/dicts/events/death/general.json]: snuck

File: resources/dicts/events/death/general.json [hunk 2]
```diff
@@ -1576,7 +1576,7 @@
         ],
         "tags": [ "no_body", "all_lives" ],
         "weight": 20,
-        "event_text": "m_c couldn't miss today! Such a bright day, perfect for a walk. That was until m_c detected another scent, a foul one. {PRONOUN/m_c/subject/CAP} recoiled, but not before long claws slashed across {PRONOUN/m_c/poss} face. Later, c_n found only blood and the scent of a predator at the scene.",
+        "event_text": "m_c met the long claws of a predator. Later, c_n found only {PRONOUN/m_c/poss} blood and a foul smell.",
```
Comment (Tiritirimatangi) [inline resources/dicts/events/death/general.json]: met their end might be better phrasing

File: resources/dicts/events/death/general.json [hunk 3]
```diff
@@ -1731,7 +1731,7 @@
         "sub_type": [ "murder" ],
         "tags": [ "all_lives" ],
         "weight": 20,
-        "event_text": "A yowl took m_c by surprise as {PRONOUN/m_c/subject} {VERB/m_c/were/was} patrolling the territory. r_c crashed into view, meowing about how m_c had to come and see. m_c followed the distressed warrior to a small ravine. Suddenly, {PRONOUN/m_c/subject} felt a sharp shove from behind, letting out a yowl of surprise that was cut short as {PRONOUN/m_c/subject} landed on the sharp rocks below.",
+        "event_text": "m_c was shoved into a ravine by r_c, having been lured near by trickery.",
```
Comment (Tiritirimatangi) [inline resources/dicts/events/death/general.json]: Lured by trickery and deceptive words, m_c was shoved into a ravine by r_c, killing them.

File: resources/dicts/events/death/general.json [hunk 4]
```diff
@@ -1785,7 +1785,7 @@
         "sub_type": ["murder"],
         "tags": ["all_lives"],
         "weight": 20,
-        "event_text": "m_c agreed to venture with r_c out onto a frozen pond. m_c went first while r_c hung back. m_c slid around playfully, taunting the other cat to chase {PRONOUN/m_c/object}. {PRONOUN/m_c/subject/CAP} {VERB/m_c/were/was} about to launch another teasing remark when the ice cracked below {PRONOUN/m_c/poss} paws. r_c lingered to make sure {PRONOUN/m_c/subject} didn't resurface before running back to camp with a yowl.",
+        "event_text": "r_c tricked m_c into playing on thin ice. {PRONOUN/r_c/subject/CAP} watched m_c disappear beneath the surface.",
```
Comment (Tiritirimatangi) [inline resources/dicts/events/death/general.json]: {PRONOUN/r_c/subject/CAP} watched m_c disappear beneath the surface, all trace of play wiped from their calculating expression.",

File: resources/dicts/events/death/general.json [hunk 5]
```diff
@@ -1849,7 +1849,7 @@
         ],
         "tags": ["all_lives"],
         "weight": 20,
-        "event_text": "m_c and r_c talked each other into walking onto the ice of the frozen pond that lay not far from camp. It was a dare at first, but when m_c didn't immediately fall in, r_c joined {PRONOUN/m_c/object}. The two cats playfully slipped around for a bit before the faint cracks reached their ears. By then, it was too late to move.",
+        "event_text": "m_c and r_c went to play together on the frozen pond. They never returned.",
```
Comment (Tiritirimatangi) [inline resources/dicts/events/death/general.json]: Last c_n heard, m_c and r_c were going out to play. They don't return, and the ice on a nearby pond is cracked...

File: resources/dicts/events/death/general.json [hunk 6]
```diff
@@ -2093,7 +2093,7 @@
         "sub_type": [ "war" ],
         "tags": [],
         "weight": 20,
-        "event_text": "In the chaos of battle and the throes of a war, it's nearly impossible to tell friend from foe. Blinded by blood, r_c accidentally kills m_c.",
+        "event_text": "Blinded by blood in the midst of battle, r_c accidentally killed m_c.",
```
Comment (Tiritirimatangi) [inline resources/dicts/events/death/general.json]: Blinded by blood in the midst of battle, r_c fatally mistakes m_c for a foe.

File: resources/dicts/events/death/general.json [hunk 7]
```diff
@@ -3044,7 +2985,7 @@
         "sub_type": [ "murder" ],
         "tags": [],
         "weight": 20,
-        "event_text": "m_c excitedly follows r_c a short distance from camp to begin the promised battle training. r_c says something about how real warriors don't hold back, even in a spar. m_c nods eagerly. {PRONOUN/m_c/subject/CAP} barely {VERB/m_c/have/has} time to duck as r_c's paw whistles past {PRONOUN/m_c/poss} head. The kit stumbles backwards, tripping over {PRONOUN/m_c/poss} paws, {PRONOUN/m_c/poss} head landing hard on a rock.",
+        "event_text": "r_c invited m_c to private battle training. When m_c slipped and hit {PRONOUN/m_c/poss} head, {PRONOUN/m_c/subject} never woke back up.",
```
Comment (Tiritirimatangi) [inline resources/dicts/events/death/general.json]: unclear if murder intentional?

File: resources/dicts/events/death/general.json [hunk 8]
```diff
@@ -3155,7 +3096,7 @@
         "sub_type": [ "murder" ],
         "tags": [],
         "weight": 20,
-        "event_text": "m_c sneaks out of camp, wondering what all is out there. {PRONOUN/m_c/subject/CAP} {VERB/m_c/come/comes} to a small stream and {VERB/m_c/attempt/attempts} to leap across. {PRONOUN/m_c/subject/CAP} misjudged the distance and {PRONOUN/m_c/poss} paws scrabble on the bank for purchase. m_c is losing {PRONOUN/m_c/poss} grip when {PRONOUN/m_c/subject} {VERB/m_c/think/thinks} {PRONOUN/m_c/subject} {VERB/m_c/catch/catches} a glimpse of r_c's pelt. {PRONOUN/m_c/subject/CAP}{VERB/m_c/'re/'s} about to call for help when {PRONOUN/m_c/poss} grip fails.",
+        "event_text": "m_c sneaked out of camp and misjudged the jump across a stream. r_c watched as {PRONOUN/m_c/subject} disappeared under the water.",
```
Comment (Tiritirimatangi) [inline resources/dicts/events/death/general.json]: snuck

Also unclear if r_c caused the death or just witnessed it

File: resources/dicts/events/death/general.json [hunk 9]
```diff
@@ -5005,7 +4946,7 @@
         "sub_type": [ "murder" ],
         "tags": [ "some_lives", "mid_lives"],
         "weight": 40,
-        "event_text": "m_c was murdered by r_c, intending to take {PRONOUN/m_c/poss} place, but r_c miscalculated how many lives m_c had left! m_c retaliated by killing r_c in self defense.",
+        "event_text": "r_c attempted to murder m_c, but miscalculated how many lives {PRONOUN/m_c/subject} had. m_c retaliated with a killing blow.",
```
Comment (Tiritirimatangi) [inline resources/dicts/events/death/general.json]: hella fun event 10/10

File: resources/dicts/events/death/general.json [hunk 10]
```diff
@@ -5144,7 +5085,7 @@
             "kit_manipulated"
         ],
         "weight": 20,
-        "event_text": "m_c saves one of the Clan's kits from drowning, but loses a life in the process. The kit is well scolded for straying into the water but is even more subdued than expected afterwards; sending nervous glances at r_c that no one else seems to catch.",
+        "event_text": "m_c lost a life while saving a kit from drowning. The kit seemed wary of r_c afterwards, but no one noticed.",
```
Comment (Tiritirimatangi) [inline resources/dicts/events/death/general.json]: notices might need to be in present tense?

File: resources/dicts/events/death/general.json [hunk 11]
```diff
@@ -5426,7 +5328,7 @@
         "sub_type": [ "murder" ],
         "tags": [],
         "weight": 20,
-        "event_text": "m_c dies during battle training with r_c, r_c exclaims that it was all an accident.",
+        "event_text": "m_c died during battle training with r_c, r_c claimed that it was all an accident.",
```
Comment (Tiritirimatangi) [inline resources/dicts/events/death/general.json]: claimed is probably too suspicious of a descriptor here

File: resources/dicts/events/death/general.json [hunk 12]
```diff
@@ -5980,7 +5882,7 @@
     "sub_type": ["murder"],
     "tags": ["lives_remain"],
     "weight": 30,
-      "event_text": "r_c attempts to take what remains of m_c's lives, but a warning from StarClan gives {PRONOUN/m_c/object} enough information to avoid such a fate. {PRONOUN/m_c/subject/CAP} still {VERB/m_c/lose/loses} a life, but at least {PRONOUN/m_c/subject} {VERB/m_c/manage/manages} to take r_c down with {PRONOUN/m_c/object}.",
+      "event_text": "A warning from StarClan gave m_c enough knowledge to survive r_c's attack. r_c could not say the same.",
```
Comment (Tiritirimatangi) [inline resources/dicts/events/death/general.json]: great phrasing here love it

File: resources/dicts/events/death/mountainous.json [hunk 13]
```diff
@@ -27,7 +27,7 @@
         "sub_type": [],
         "tags": ["clan:kitten"],
         "weight": 20,
-        "event_text": "m_c sneaks out of camp but finds {PRONOUN/m_c/self} lost in the strange, angular tunnels that lead out into the territory. The Clan searches for {PRONOUN/m_c/object} in the morning to no avail. The leftover scent of blood and mountain weasel serves as an intense warning to the other kits.",
+        "event_text": "m_c snuck out of camp and was lost to the strange tunnels, nothing left to be found but blood and weasel scent.",
```
Comment (Tiritirimatangi) [inline resources/dicts/events/death/mountainous.json]: nothing left to find but

**Review by Tiritirimatangi — CHANGES_REQUESTED** [review]
This work is much needed and MUCH appreciated!

File: resources/dicts/events/death/general.json (hunk 1 above)
Comment (SunlitFable) [inline resources/dicts/events/death/general.json]: just for readability i might recommend a couple edits;

> intending to find **out** more about {PRONOUN/m_c/poss} origins

or

> intending to **discover** more about {PRONOUN/m_c/poss} origins

‎ 
this one is optional but might be good;
> **After a long search in the morning,** c_n found {PRONOUN/m_c/poss} cooling body.

Comment (SunlitFable) [reply resources/dicts/events/death/general.json]: both sneaked and snuck are correct actually!

File: resources/dicts/events/death/general.json [hunk 14]
```diff
@@ -4471,7 +4412,7 @@
         "sub_type": [ "old_age" ],
         "tags": [ "all_lives" ],
         "weight": 20,
-        "event_text": "m_c's body just couldn't continue forward - {PRONOUN/m_c/subject}{VERB/m_c/'ve/'s} lost all of {PRONOUN/m_c/poss} remaining lives to old age.",
+        "event_text": "{PRONOUN/m_c/subject}{VERB/m_c/'ve/'s} lost all of {PRONOUN/m_c/poss} remaining lives to old age.",
```
Comment (SunlitFable) [inline resources/dicts/events/death/general.json]: this only has pronouns, no mention of the leader's name

File: resources/dicts/events/death/general.json [hunk 15]
```diff
@@ -4851,7 +4792,7 @@
             "all_lives"
         ],
         "weight": 20,
-        "event_text": "Despite StarClan's efforts, m_c's body just couldn't continue. {PRONOUN/m_c/poss/CAP} remaining lives were taken by old age.",
+        "event_text": "{PRONOUN/m_c/poss/CAP} remaining lives were taken by old age.",
```
Comment (SunlitFable) [inline resources/dicts/events/death/general.json]: also just pronouns and no name

File: resources/dicts/events/death/general.json [hunk 16]
```diff
@@ -5082,7 +5023,7 @@
             "all_lives"
         ],
         "weight": 20,
-        "event_text": "m_c and r_c were on a walk together when a pack of dogs caught them off guard. m_c prepared to fight, but when {PRONOUN/m_c/subject} looked to r_c for backup, the other cat was nowhere to be found. r_c watched quietly as the dogs took life after life from m_c.",
+        "event_text": "m_c and r_c were ambushed by a pack of dogs. r_c was nowhere to be found when m_c looked for help.",
```
Comment (SunlitFable) [inline resources/dicts/events/death/general.json]: sentences feel stilted and short; mayhaps separate with a comma instead?

> m_c and r_c were ambushed by a pack of dogs, but r_c was nowhere to be found when m_c looked for help.

File: resources/dicts/events/death/mountainous.json (hunk 13 above)
Comment (SunlitFable) [inline resources/dicts/events/death/mountainous.json]: a small nitpick, but "the strange tunnels" makes it sound as though it's a known location though it hasn't been mentioned before. maybe something like "the winding tunnels" instead?

File: resources/dicts/events/death/general.json (hunk 1 above)
Comment (scribblecrumb) [reply resources/dicts/events/death/general.json]: I despise the word sneaked but also I keep hearing it's more grammatically correct (and pycharm flags snuck as a typo) lmao might be worth a larger style discussion if we want to be consistent with the term clangen uses?

File: resources/dicts/events/death/mountainous.json (hunk 13 above)
Comment (scribblecrumb) [reply resources/dicts/events/death/mountainous.json]: hmm I don't mind changing it; but for added context this is a Ruins specific event and is meant to reference the idea that the entrance and exit to the camp is the ruined doorway pictured on the BG.  Essentially "the strange tunnels" are the halls of the ruins that the cats must traverse.  I don't think "winding" quite gives the same vibe, is there a different way we could imply the intention?

File: resources/dicts/events/death/general.json (hunk 2 above)
Comment (scribblecrumb) [reply resources/dicts/events/death/general.json]: hmm counterpoint: I wanted to preserve the original "long claws slashed across their face" vibes

File: resources/dicts/events/death/general.json (hunk 4 above)
Comment (scribblecrumb) [reply resources/dicts/events/death/general.json]: I feel like that lengthens it a bit too much, but I recognize that my version is a bit vague.  Perhaps something like 
```suggestion
        "event_text": "r_c tricked m_c into playing on thin ice. {PRONOUN/r_c/subject/CAP} watched closely as m_c disappeared beneath the surface.",
```
would work? It implies more intent to ensure m_c goes down.

File: resources/dicts/events/death/general.json (hunk 7 above)
Comment (scribblecrumb) [reply resources/dicts/events/death/general.json]: Original event is also pretty unclear in that regard... perhaps we could workshop it in the discord?

File: resources/dicts/events/death/general.json (hunk 8 above)
Comment (scribblecrumb) [reply resources/dicts/events/death/general.json]: Same for the OG event, but we could also workshop this one more if we want it to be more clear?

File: resources/dicts/events/death/general.json [hunk 17]
```diff
@@ -5426,7 +5328,7 @@
         "sub_type": [ "murder" ],
         "tags": [],
         "weight": 20,
-        "event_text": "m_c dies during battle training with r_c, r_c exclaims that it was all an accident.",
+        "event_text": "m_c died during battle training with r_c, r_c exclaimed that it was all an accident.",
```
Comment (selkirks) [inline resources/dicts/events/death/general.json]: I'd add a period after the first r_c and replace the second r_c with a pronoun tag.

File: resources/dicts/events/death/mountainous.json [hunk 18]
```diff
@@ -27,7 +27,7 @@
         "sub_type": [],
         "tags": ["clan:kitten"],
         "weight": 20,
-        "event_text": "m_c sneaks out of camp but finds {PRONOUN/m_c/self} lost in the strange, angular tunnels that lead out into the territory. The Clan searches for {PRONOUN/m_c/object} in the morning to no avail. The leftover scent of blood and mountain weasel serves as an intense warning to the other kits.",
+        "event_text": "m_c snuck out of camp and was lost to the strange tunnels, nothing left to find but blood and weasel scent.",
```
Comment (selkirks) [inline resources/dicts/events/death/mountainous.json]: raccoon-scent used a dash in an earlier death outcome. weasel scent should be dashed as well for consistency

**Review by selkirks — DISMISSED** [review]

File: resources/dicts/events/death/general.json (hunk 2 above)
Comment (SunlitFable) [reply resources/dicts/events/death/general.json]: "met their end to the long claws of a predator" purrhaps?

File: resources/dicts/events/death/mountainous.json (hunk 13 above)
Comment (SunlitFable) [reply resources/dicts/events/death/mountainous.json]: oh gotcha, i missed that! hrm... is there a reason we can't add "angular" back into that description?

Comment (scribblecrumb) [reply resources/dicts/events/death/mountainous.json]: I spose not, I removed it for shortening purposes but one additional word isn't bad lol

File: resources/dicts/events/death/general.json (hunk 2 above)
Comment (scribblecrumb) [reply resources/dicts/events/death/general.json]: Ooh, I like that

File: resources/dicts/events/death/general.json (hunk 1 above)
Comment (Tiritirimatangi) [reply resources/dicts/events/death/general.json]: I think this is a learned/learnt problem aka it's not our problem that english is Like This and we can go with either whenever

File: resources/dicts/events/death/general.json (hunk 4 above)
Comment (Tiritirimatangi) [reply resources/dicts/events/death/general.json]: Love it, perf!

File: resources/dicts/events/death/general.json (hunk 7 above)
Comment (Tiritirimatangi) [reply resources/dicts/events/death/general.json]: I think so, if it's hard to workshop we could just cut this individual event

File: resources/dicts/events/death/general.json (hunk 8 above)
Comment (Tiritirimatangi) [reply resources/dicts/events/death/general.json]: I think 'watched without concern' would be enough to tip it into murder territory

Comment (scribblecrumb) [reply resources/dicts/events/death/general.json]: oooh, I like that! Yeah we can go with that

**Review by Tiritirimatangi — APPROVED** [review]

**Pull request merged**